src/FellowOakDicom.Serialization.Forked/FellowOakDicom.Serialization.Forked.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Maybe `Microsoft.FellowOakDicom.Serialization` instead? Or perhaps something else? I am not a big fan of the namespace `Forked` lol